features/networking/multus.feature

```diff
@@ -255,12 +231,168 @@ Feature: Multus-CNI related scenarios
+    And the output should contain "net2"
+    And the output should contain "macvlan mode bridge"
+    And the output should contain "macvlan mode private"
+    When I execute on the pod:
+      | bash | -c | /usr/sbin/ip -f inet addr show net2 \| grep -Po 'inet \K[\d.]+' |
+    Then the output should match "10.1.1.\d{1,3}"
+    And the expression should be true> IPAddr.new(@result[:response].chomp)
+
+  # @author bmeng@redhat.com
+  # @case_id OCP-21859
+  @admin
+  @destructive
+  Scenario: Create pods with muliple cni plugins via multus-cni - host-device + host-device
+    # Make sure that the multus is enabled
+    Given the master version >= "4.0"
+    And the multus is enabled on the cluster
+    And I select a random node's host
+    And evaluation of `node.name` is stored in the :target_node clipboard
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/host-device.yaml |
+      | n | <%= project.name %> |
+    Then the step should succeed
+    When I run oc create as admin over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/NetworkAttachmentDefinitions/host-device.yaml" replacing paths:
+      | ["metadata"]["name"]      | host-device-2       |
+      | ["spec"]["config"]        | '{"cniVersion": "0.3.0", "type": "host-device", "device": "eth2008"}' |
+      | ["metadata"]["namespace"] | <%= project.name %> |
+    Then the step should succeed
+
+    # Prepare the net link on the node which will be attached to the pod
+    When I run network command on the "<%= cb.target_node %>" node's sdn pod:
+       | sh | -c | ip link add eth08 link eth0 type macvlan mode bridge |
+    Then the step should succeed
+    When I run network command on the "<%= cb.target_node %>" node's sdn pod:
+       | sh | -c | ip link add eth2008 link eth0 type macvlan mode bridge |
+    Then the step should succeed
+    Given I register clean-up steps:
+    """
+    I run network command on the "<%= cb.target_node %>" node's sdn pod:
+       | sh | -c | ip link del eth08 |
+    the step should succeed
+    I run network command on the "<%= cb.target_node %>" node's sdn pod:
+       | sh | -c | ip link del eth2008 |
+    the step should succeed
+    """
+
+    # Create the pod which consumes two hostdev custom resources
+    When I run oc create over "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/multus-cni/Pods/2interface-hostdevice-hostdevice.yaml" replacing paths:
+      | ["spec"]["nodeName"] | "<%= cb.target_node %>" |
+      | ["metadata"]["annotations"]["k8s.v1.cni.cncf.io/networks"] | host-device, host-device-2 |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | name=two-host-device-pod |
+    Given I register clean-up steps:
```

Comment:
here delete the pod means: it's high priority when entry the clean up flow after the scenario?

Reply:
We need to delete the pod to release the host-device to the host, and then delete the net link from the host. So delete the pod has higher priority.

Reply:
Here also I think clean-up should register before we check pod is ready.